releasenotes/notes/c++vartype-info-ae0bf7b4ca0a82bd.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    Add ``dimod::vartype_limits`` to C++ code for compile-time information
+    about variable types.
+  - |
+    Add ``dimod::vartype_info`` to C++ code for runtime information
+    about variable types.
+  - |
+    Add ``vartype_info()`` function.
```

Comment:
no newline at EOF here

Reply:
good catch, but since this is a release note file we'll never touch it again (since it's position in the git history is how reno determines when it was released) so it doesn't matter